playbooks/common/openshift-cluster/openshift_hosted.yml

```diff
@@ -47,7 +47,7 @@
     openshift_hosted_logging_elasticsearch_ops_pvc_prefix: "{{ 'logging-es-ops' if openshift_hosted_loggingops_storage_kind | default(none) =='dynamic' else '' }}"
 
   - role: cockpit-ui
-    when: ( openshift.common.version_gte_3_3_or_1_3  | bool ) and ( openshift_hosted_manage_registry | default(true) | bool ) and not (openshift.docker.hosted_registry_insecure | default(false) | bool)
+    when: ( openshift.common.version_gte_3_3_or_1_3  | bool ) and ( openshift_hosted_manage_registry | default(true) | bool ) and not (openshift_docker_hosted_registry_insecure | default(false) | bool)
```

Comment:
Eventually we should change these to be multiline ... doesn't have to happen in this PR.